### ISSUE TITLE
terminology

### ISSUE THREAD

**Issue by pdarragh** [Radmind/radmind#258]
I think the term dektop-negative is misleading, as someone using laptops 
might think it doesn't apply to them.

Maybe staff/kiosk, or loose/tight, i don't know. Can you think of a better 
term?

Also, the main 3 options in first run are a bit wordy and if we really want 
to be picky - you should not use a comm after "and".

Maybe simplify the wording with something like:

Setup Radmind Server
Setup Radmind Client
Update Radmind Client

Let me know what you think and I'll change the working in the manual.

Thanks

Ofir

Original comment by: ofirgal

**Comment by linuxjackto1** [comment]
thanks www.movied.org
